### DISCUSSION TITLE
如何把帧数限制到30帧

### DISCUSSION THREAD

**Discussion by Shadowfish123** [Blinue/Magpie#842] [Q&A] [answered]
超分辨率玩galgame的时候，60帧，显卡占用达到70%，风扇一直在响。实际上玩galgame30帧就够了，帧数再高也是浪费性能，通过n卡控制面板调的话玩3a的时候又得调回去，很麻烦。有没有办法直接在Magpie里限制帧数。

**Comment by Blinue** [comment, marked as answer]
#755

**Comment by mrl998** [comment]
你可以在N卡面板里给magpie限帧
